### DISCUSSION TITLE
Just Saying Hello

### DISCUSSION THREAD

**Discussion by Dorane-Star** [Cyfrin/foundry-full-course-cu#3020] [General]
Joined recently.........and thought of saying Hi, and figuring out how the discussion tab looks like.

**Comment by EngrPips** [comment]
Welcome aboard, We wish you the best.

**Comment by Dorane-Star** [comment]
thanks a lot

**Comment by onlyoneAlexia** [comment]
hello

**Reply by EngrPips** [reply]
Hello, Friend. We are glad to have you.
